CHANGES.rst

```diff
@@ -19,6 +24,7 @@ Bugfixes
 - Fix error in HTTP API when Basic auth headers are present (:issue:`4123`,
   thanks :user:`uxmaster`)
 
+
```

Comment:
we usually just have one blank line between those versions, so please don't add another one here ;)

Reply:
fixed